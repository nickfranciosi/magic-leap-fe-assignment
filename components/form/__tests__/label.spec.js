/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import Label from '../label';

describe('Price', () => {
  test('it should render', () => {
    const tree = renderer.create(<Label />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('it should be uppercase', () => {
    const tree = renderer.create(<Label />).toJSON();
    expect(tree).toHaveStyleRule('text-transform', 'uppercase');
  });
});
