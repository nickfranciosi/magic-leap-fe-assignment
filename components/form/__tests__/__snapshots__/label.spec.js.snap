// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Price it should render 1`] = `
.c0 {
  color: #edec51;
  text-transform: uppercase;
  font-size: 11px;
  margin-bottom: 4px;
  font-family: 'Lato',Helvetica,arial,sans-serif;
  -webkit-letter-spacing: -1.1px;
  -moz-letter-spacing: -1.1px;
  -ms-letter-spacing: -1.1px;
  letter-spacing: -1.1px;
}

<label
  className="c0"
/>
`;
